client/src/pages/Login.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./LoginStyles.css";
+import Layout from "../components/Layout";
+import Button from "../components/Button";
+
+function Login() {
+  const handleClick = () => {
```

Comment:
these clickhandlers should be named so that it's easier to understand what they're doing.

In these cases it could be something like `requestMagicLink` and `createAccount`